test/xdpmp/miniport.c

```diff
@@ -1062,24 +1072,33 @@ MpSetOffloadAttributes(
 {
     NDIS_MINIPORT_ADAPTER_OFFLOAD_ATTRIBUTES OffloadAttributes = {0};
     NDIS_OFFLOAD Offload = {0};
+    const ULONG Encapsulation = NDIS_ENCAPSULATION_IEEE_802_3;
```

Comment:
Why add this extra `const` instead of using the `#define` directly below?

Reply:
1. this reduces line length, making the code more readable
2. we actually may want to support more encapsulations in the future. this makes that maintenance easier